src/main/java/TaskList.java

```diff
@@ -0,0 +1,160 @@
+import util.Formatter;
+import util.Storage;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+import static util.Parser.getArgMap;
+import static util.Parser.getCommand;
+
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    private String taskCountMsg() {
```

Comment:
Would `getTaskCountMsg()` be a better name to make it consistent with the other method's namings